AEPCampaignClassic/Sources/Event+CampaignClassic.swift

```diff
@@ -10,8 +10,59 @@
  */
 
 import AEPCore
+import AEPServices
 import Foundation
 
 extension Event {
 
+    /// Returns true if this event is a Campaign Classic register event
+    var isRegisterEvent: Bool {
+        return data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.REGISTER_DEVICE] as? Bool ?? false
+    }
+
+    /// Returns true if this event is a Campaign Classic TrackNotificationClick event
+    var isTrackClickEvent: Bool {
+        return data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.TRACK_CLICK] as? Bool ?? false
+    }
+
+    /// Returns true if this event is a Campaign Classic TrackNotificationReceive event
+    var isTrackReceiveEvent: Bool {
+        return data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.TRACK_RECEIVE] as? Bool ?? false
+    }
+
+    /// Retrieves the broadlogId string from the event data if available, nil otherwise
+    var broadlogId: String? {
+        if let trackingInfo = data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.TRACK_INFO] as? [String: Any] {
```

Comment:
nit: can add private var for `trackingInfo` so you don't have to repeat this code for multiple getters